news/PR964.dev

```diff
@@ -0,0 +1 @@
+Add guidelines for admins on release-monitoring.org
```

Comment:
Let's rather got with `PR964.other` instead of dev change.

Reply:
sorry, didn't read closely enough. fixed.